python_modules/dagster/dagster/daemon/workspace.py

```diff
@@ -68,15 +71,36 @@ def get_repository_location(self, location_name: str) -> RepositoryLocation:
 
         return location_entry.repository_location
 
-    def cleanup(self) -> None:
+    # change this default param
```

Comment:
i think i had been considering taking out the default param and making callsites set this explicitly

Reply:
Resolved in https://github.com/dagster-io/dagster/pull/8642